### ISSUE TITLE
Finish Storm Specification UI

### ISSUE THREAD

**Issue by arnelmandilag** [besasm/EMGAATS#191]
orms with actual and virtual gages

BTS admin privs to \\besfile1\modeling\SAMaster_22\rainfall
programatically create list of design storms. 

__CORE TEAM DISCUSSION__
config vs database storage of design storm

**Comment by arnelmandilag** [comment]
Finished main feature of using Neptune specifications for actual/virtual gages

Currently the selection of actual and virtual gages are automatic. Possible enhancements include
- Selection of which actual gages to use
- Selection of quartersections for virtual gages. Currently all draining nodes get their own virtual gage.
- Providing a progress bar as the raingage file creation can take a while
- Note that EPA SWMM 5 can take a while reading a long rainfall file (long multiday period + short timestep +  large number of gages)

@collinsfoobar: The DeployModel's click has been changed from the test version included in InternalOnClick() to a dynamic version called InternalOnClick2() that changes depending on the simulation being deployed.

While finishing off this task, I think I finally ran into the questions you were trying to get answered before: there is no actual Model object sitting anywhere. I had to create one on the fly in InternalOnClick2(), and it merely ties in the Network sitting in EmgaatsExtension.Instance.NetworkManager.Network and fills the subcatchments and hydraulics structures (simlinks, etc). This then allows the deployer object (of type DeploySWMM5) to work. But other than this rather contrived way of getting deplyoment out, it seems the Model object doesn't really do anything significant yet. Simulation objects may be better off just having direct access to the Network object, as it is already implied, rather than accessing it via a Model object.
